linode/instanceconfig/schema_resource.go

```diff
@@ -18,16 +18,27 @@ var resourceSchema = map[string]*schema.Schema{
 	"label": {
 		Type:        schema.TypeString,
 		Required:    true,
-		Description: "The Config’s label for display purposes only.",
+		Description: "The Config's label for display purposes only.",
+	},
+
+	"device": {
+		Type:          schema.TypeList,
+		Elem:          &schema.Resource{Schema: deviceV2Schema},
+		Optional:      true,
+		Computed:      true,
+		ConflictsWith: []string{"devices"},
+		Description:   "A dictionary of device disks to use as a device map in a Linode's configuration profile.",
```

Comment:
This description seems to fit `device_map` rather than `device`. 

Maybe something like this:

```
A list of assignments between disks and devices for this configuration profile.
```